setup.cfg

```diff
@@ -0,0 +1,2 @@
+[install]
```

Comment:
What is this file about?

Reply:
I just copied it from your installation. Because your files were missing the `plist` file I installed the release from the zipped file and copied the missing files.